onnxruntime/core/framework/allocation_planner.cc

```diff
@@ -990,6 +990,7 @@ class PlannerImpl {
     return true;
   }
 
+#ifndef ORT_MINIMAL_BUILD
```

Comment:
also need to exclude from minimal extended build

Reply:
Done.